coop-budget/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "coop-budget"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bytes = "1.4.0"
+hdrhistogram = "7.5.2"
+tokio = { version = "1", path = "../tokio", features = ["full", "tracing"] }
+tokio-util = { version = "0.7.8", path = "../tokio-util", features = ["codec"] }
+rand = "0.8.4"
+uuid = { version = "1.3.3", features = ["v4", "serde"] }
+tracing = "0.1.37"
+tracing-subscriber = { version = "0.3.17", features = ["env-filter", "fmt"] }
+anyhow = "1.0.40"
```

Comment:
The minrust check is probably failing due to one of these dependencies.

Reply:
very strange, I remove the whole coop-budget folder, and run below in my machine
 ```
gftea@E-CND:~/repo/tokio$ cargo +1.56.0 check --workspace --all-features
    Updating crates.io index
error: failed to select a version for the requirement `aho-corasick = "=1.0.2"`
candidate versions found which didn't match: 0.7.20, 0.7.19, 0.7.18, ...
location searched: crates.io index
required by package `regex v1.8.4`
    ... which satisfies dependency `regex = "=1.8.4"` of package `predicates v2.1.1`
    ... which satisfies dependency `predicates = "=2.1.1"` of package `mockall v0.11.1`
    ... which satisfies dependency `mockall = "=0.11.1"` of package `tokio v1.28.2 (/home/gftea/repo/tokio/tokio)`
    ... which satisfies path dependency `tokio` of package `benches v0.0.0 (/home/gftea/repo/tokio/benches)`
    ```